receiver/vcenterreceiver/README.md

```diff
@@ -55,4 +55,7 @@ The full list of settings exposed for this receiver are documented [here](./conf
 
 Details about the metrics produced by this receiver can be found in [metadata.yaml](./metadata.yaml) with further documentation in [documentation.md](./documentation.md)
 
+Additionally some metrics are only available to certain vSphere API versions.
+- [vcenter.vm.cpu.readiness](https://dp-downloads.broadcom.com/api-content/apis/API_VMA_001/8.0U2/html/vim.vm.Summary.QuickStats.html)
```

Comment:
Small thing, but I think having `7.0.0+` as a header for this list would be appropriate.

Reply:
I'm not sure how accurate [this page](https://endoflife.date/vcenter) is, but it claims that 6.7 is EOL, so perhaps we should just drop support?

Reply:
Yeah, it seems to be accurate since there is also an update from [Vmware](https://core.vmware.com/blog/reminder-vsphere-6567-end-general-support) saying the same thing.